src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx

```diff
@@ -42,11 +42,13 @@ const HarvestAction: React.FunctionComponent<FarmWithStakedValue> = ({ pid, user
     updateValue.current(earningsBusd)
   }, [earningsBusd, updateValue])
 
+  const tokenEarnedString = TranslateString(330, 'CAKE Earned', { asset: 'CAKE' }).split(' ')
```

Comment:
cake doesn't need to be translated, and this split can lead to unexpected behaviour. Please remove and just translate `EARNED`

Reply:
Done